### PR TITLE
fix: Updated the get-started codelab

### PR DESCRIPTION
**Pull request by albertodev01**
_Description of what this PR is changing or adding, and why:_ Here we are with another try, hopefully the correct one!

Please @domesticmouse @khanhnwin help me in this PR is anything is wrong. You merged https://github.com/flutter/codelabs/pull/279 so the examples are now updated. I've updated submodules and the example itself

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by albertodev01** [comment]
Please merge this (https://github.com/flutter/codelabs/pull/280) because I need the new docregions.

After you've merged that, I will update the submodule, run the tool to refresh code excerpts and then we should hopefully be done with this

File: src/get-started/codelab.md [hunk 1]
```diff
@@ -571,9 +571,8 @@ lazily, on demand.
       }
     ```
 
- 4. In the `_RandomWordsState` class, update the `build()` method to use
-    `_buildSuggestions()`, rather than directly calling the word
-    generation library.  ([`Scaffold`][]
+ 4. We not have the `_RandomWordsState` class using a `ListView.builder`
+    rather than directly calling the word generation library.  ([`Scaffold`][]
```
Comment (sfshaza2) [inline src/get-started/codelab.md:575]: @albertodev01, I don't understand this sentence. Did you mean "We DON'T have..." But, even if you did, you have removed a sentence that provided an instruction on what to do, with a sentence that does not tell the user what to do.

**Review by sfshaza2 — COMMENTED** [review]
This PR appears to be failing the check against the stable channel. But I do have one piece of feedback that needs to change.

**Comment by domesticmouse** [comment]
@albertodev01 the linked PR is landed, please update

File: src/get-started/codelab.md (hunk 1 above)
Comment (albertodev01) [reply src/get-started/codelab.md:575]: I made a typo! I meant "We now have" instead of "We not have"

**Review by domesticmouse — APPROVED** [review]

**Comment by albertodev01** [comment]
I am updating code excerpts on my macOS machine but the `update-code-excerpts.sh` tool is really broken on windows. On macOS it works out of the box!

I will open an issue

**Comment by alberto-sf** [comment]
Screenshot 2022-02-16 at 01 50 30

On my mac machine I'm running the script but CI still complains. How can I see where the problem is? The only affected file is the markdown one, and I honestly have no clue on what to do next

Screenshot 2022-02-16 at 01 52 33

**Comment by domesticmouse** [comment]
@albertodev01 I've re-run the `refresh-code-excerpts.sh` script and pushed the changes to https://github.com/flutter/website/pull/6817/commits/cc50bc99bc7d80de08ef422bc868e75e4ba93bff

Let me know if that looks correct to you, or if we need another revision on `flutter/codelabs` to get the snippets you are looking for.

**Comment by albertodev01** [comment]
@domesticmouse @sfshaza2 looks good to me! We can finally merge I guess 🙂

**Comment by domesticmouse** [comment]
Ptal @sfshaza2

**Comment by khanhnwin** [comment]
Yes!! Thank you for sticking with this one @albertodev01!!

**Comment by albertodev01** [comment]
@khanhnwin could you also approve?

**Comment by sfshaza2** [comment]
WOOT! She has landed.

**Pull request merged**